test/choice_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class ChoiceTest < MiniTest::Test
+  def setup
+    @choice = Chiketto::Choice.new 'answer' => {
+        'text' => 'test choice',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.